styles/style.css

```diff
@@ -6,3 +6,127 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  font-family: sans-serif;
+}
+
+nav {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.spotifylogo {
+  width: 150px;
+  height: 50px;
+  display: flex;
```

Comment:
spotifylogo is a child element of your nav flex container, so you don't need to add display: flex and justify-content -- these are properties only available to the parent container.